### DISCUSSION TITLE
How to fetch data from strongRef

### DISCUSSION THREAD

**Discussion by fxsobr** [MarshalX/atproto#369] [Questions]
Notification = models.AppBskyNotificationListNotifications.Notification
 
When a user tags me, I'm trying to get the name and text of the original post and not the reply. How did I get this data, considering that when I take a print of the Notification I only have the uri of the original post, how did I get these values?

Thanks

**Comment by MarshalX** [comment]
solved? if so it could be better to provide answer to yourself. it could help the others with the same question :)
